docs/openj9_support.md

```diff
@@ -228,6 +229,46 @@ OpenJDK 21 and later binaries are expected to function on the minimum operating
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Not all of these distributions are tested, but the following distributions are expected to function without problems:
+
+- Linux distributions that have a minimum glibc version 2.17
+
+| Windows                                   | x64                                                                                  |
+|-------------------------------------------|--------------------------------------------------------------------------------------|
+| Windows 10                                | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| Windows 11                                | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| Windows Server 2016                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| Windows Server 2019                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| Windows Server 2022                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+| macOS         | x64                                                                                  | AArch64 (Apple silicon)                                                              |
+|---------------|--------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------:|
+| macOS 13      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| macOS 14      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| macOS 15      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+
+| AIX                                       | ppc64                                                                                |
+|-------------------------------------------|--------------------------------------------------------------------------------------|
+| AIX 7.2 TL5                               | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+:fontawesome-solid-bell:{: .warn aria-hidden="true"} **Important:** From the 0.55.0 release onwards, AIX OpenJ9 builds require the [XL C++ Runtime 17.1.3.0 or later](https://www.ibm.com/support/pages/fix-list-xl-cc-runtime-aix#17130).
```

Comment:
This shouldn't refer to the 0.55 release. It's a statement about jdk25. i.e. For OpenJDK 25 and later binaries, AIX OpenJ9 builds require the [XL C++ Runtime 17.1.3.0 or later].

Reply:
In the [earlier comment](https://github.com/eclipse-openj9/openj9-docs/pull/1570#discussion_r2296973660), you have said that xlc/C++ should stay as 16.1.0.20 for the 0.55 release. But, this statement should still refer to 17.1.3.0 or later? i.e. For OpenJDK 25 and later binaries, AIX OpenJ9 builds require the [XL C++ Runtime 17.1.3.0 or later]?

Reply:
That's correct. We want a 17.1.3.0 minimum requirement from GA since the compiler is expected to change in a later release.